src/main/scala/extractors/Extractor.scala

```diff
@@ -0,0 +1,44 @@
+package extractors
+
+object Extractor {
+
+  val inUaStr: String = "in.ua"
+  val gmailComStr: String = "gmail.com"
+
+  def filterNotDomainsWithParticularHead(domains: List[String], head: String = inUaStr): List[String] = {
+    domains.filterNot {
+      case Domain(f, s, _*) if s"$s.$f" == head => true
+      case _ => false
+    }
+  }
+
+  def filterEmailsWithParticularDomain(emails: Array[String], inputDomainStr: String = gmailComStr): Array[Unit] = {
```

Comment:
Please, return `Unit` instead of `Array[Unit]`. Array[Unit] - is confusing at least.

Reply:
Changed to Unit and committed